coders/webp.c

```diff
@@ -232,11 +232,21 @@ static int FillBasicWEBPInfo(Image *image,const uint8_t *stream,size_t length,
 }
 
 static int ReadSingleWEBPImage(Image *image,const uint8_t *stream,
-  size_t length,WebPDecoderConfig *configure,ExceptionInfo *exception)
+  size_t length,WebPDecoderConfig *configure,ExceptionInfo *exception,MagickBooleanType is_first)
```

Comment:
We use a line limit of 80 characters in our project. Could you move this to the next line?